docs/edge.md

```diff
@@ -115,100 +140,30 @@ Available capital doesn’t change before a position is sold. Let’s assume tha
-
-Having a win rate of 100% on a single trade doesn't mean anything at all. But having a win rate of 70% over past 100 trades means clearly something.
-
-(defaults to 10, it is highly recommended not to decrease this number)
-
-#### max_trade_duration_minute
-
-Edge will filter out trades with long duration. If a trade is profitable after 1 month, it is hard to evaluate the strategy based on it. But if most of trades are profitable and they have maximum duration of 30 minutes, then it is clearly a good sign.
-
-**NOTICE:** While configuring this value, you should take into consideration your ticker interval. As an example filtering out trades having duration less than one day for a strategy which has 4h interval does not make sense. Default value is set assuming your strategy interval is relatively small (1m or 5m, etc.).
-
-(defaults to 1 day, i.e. to 60 * 24 = 1440 minutes)
-
-#### remove_pumps
-
-Edge will remove sudden pumps in a given market while going through historical data. However, given that pumps happen very often in crypto markets, we recommend you keep this off.
+Edge module has following configuration options:
 
-(defaults to false)
+|  Parameter | Description |
+|------------|-------------|
+| `enabled` | If true, then Edge will run periodically. <br>*Defaults to `false`.* <br> ***Datatype:*** *Boolean*
+| `process_throttle_secs` | How often should Edge run in seconds. <br>*Defaults to `3600` (once per hour).* <br> ***Datatype:*** *Integer*
+| `calculate_since_number_of_days` | Number of days of data against which Edge calculates Win Rate, Risk Reward and Expectancy. <br> **Note** that it downloads historical data so increasing this number would lead to slowing down the bot. <br>*Defaults to `7` (once per hour).* <br> ***Datatype:*** *Integer*
+| `capital_available_percentage` | This is the percentage of the total capital on exchange in stake currency. <br>As an example if you have 10 ETH available in your wallet on the exchange and this value is 0.5 (which is 50%), then the bot will use a maximum amount of 5 ETH for trading and considers it as available capital. <br>*Defaults to `0.5` (once per hour).* <br> ***Datatype:*** *Float*
+| `allowed_risk` | Ratio of allowed risk per trade. <br>*Defaults to `0.01` (1%)).* <br> ***Datatype:*** *Float*
+| `stoploss_range_min` | Minimum stoploss. <br>*Defaults to `-0.01`.* <br> ***Datatype:*** *Float*
+| `stoploss_range_max` | Maximum stoploss. <br>*Defaults to `-0.10`.* <br> ***Datatype:*** *Float*
+| `stoploss_range_step` | As an example if this is set to -0.01 then Edge will test the strategy for `[-0.01, -0,02, -0,03 ..., -0.09, -0.10]` ranges. <br> **Note** than having a smaller step means having a bigger range which could lead to slow calculation. <br> If you set this parameter to -0.001, you then slow down the Edge calculation by a factor of 10. <br>*Defaults to `-0.001`.* <br> ***Datatype:*** *Float*
```

Comment:
The example should probably be changed (other value instead of -0.001 used), because "If you set this parameter to -0.001" sounds like the default value slows down the Edge calculations.

Reply:
I'm not sure about this (how to change it - if the default value is ok ...).

The old version had -0.01 as default - but it wasn't the default when looking at the code, that's why i changed it to -0.001 (to align documentation with code).

now the sample and calculation is not wrong per se - going from -0.01 to -0.001 slows down by a factor of then - the same as when going from -0.001 to -0.0001...

Reply:
Yes, I understand why you have changed it in the description.

Okay, probably the default value in the code should be aligned to one used everywhere in the sample configs (to be -0.01). Not in this PR.

Without this the phrase used here looks a bit strange -- like the default value makes the calculations slower and thus should not be used...